pkgs/top-level/all-packages.nix

```diff
@@ -2759,6 +2759,8 @@ in
 
   dvtm-unstable = callPackage ../tools/misc/dvtm/unstable.nix {};
 
+  dyndnsc = python3Packages.dyndnsc;
```

Comment:
This is not correct. Is it an application or a library or both?

Reply:
Since it should only be used as an application, I moved it to `pkgs/applications/networking/dyndns/dyndnsc` and removed it from `python-packages.nix`.